terraform/log-alert.tf

```diff
@@ -0,0 +1,30 @@
+module "logworkspace" {
```

Comment:
can you add .DS_Store to the .gitignore and remove the one that's been checked in at `terraform/.DS_Store`